test/rails_app/db/migrate/20100401102949_create_tables.rb

```diff
@@ -10,7 +10,7 @@ def self.up
 
       ## Database authenticatable
       t.string :email,              null: false, default: ""
-      t.string :encrypted_password, null: false, default: ""
+      t.string :encrypted_password, default: ""
```

Comment:
@sivagollapalli can you please revert this migration change? Having `null: false` won't replicate the production scenarios we have on existing apps - sadly we will have to tweak some other pieces of the codebase/test cases to avoid persisting nil `encrypted_password` values.